src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Linux/IFileSystem.cs

```diff
@@ -16,20 +16,20 @@ internal interface IFileSystem
     /// <summary>
     /// Checks for file existence.
     /// </summary>
-    /// <returns> True/False.</returns>
+    /// <returns><see langword="true"/> or <see langword="false"/>.</returns>
```

Comment:
The established pattern is to write as follows:
```xml
    /// <returns><see langword="true"/> if [explain the condition]; <see langword="false"/>, otherwise.</returns>
```

```suggestion
    /// <returns><see langword="true"/> if file exists; <see langword="false"/>, otherwise.</returns>
```

Furthermore, we could copy the existing docs wording, when appropriate. E.g., here's the doc for `File.Exists` API: https://learn.microsoft.com/dotnet/api/system.io.file.exists

Reply:
Done